search-model/src/settings.py

```diff
@@ -12,9 +12,13 @@
 
 preprocess_images = True
 # directory of images to resize and save to processed directory
-raw_image_dir=os.path.join(BASE_DIR, 'data','raw','eth_material_archive')
+#raw_image_dir=os.path.join(BASE_DIR, 'data','raw','eth_material_archive')
 # directory of images to use to calculate features
-processed_image_dir=os.path.join(BASE_DIR, 'data','processed','eth_material_archive')
+#processed_image_dir=os.path.join(BASE_DIR, 'data','processed','eth_material_archive')
+raw_image_dir=os.path.join(BASE_DIR, 'data_raw_ethz','images')
```

Comment:
I would keep the folder structure as "data/raw/ethz" instead of flattening it to data_raw_ethz